src/components/Temp/Temp.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
```

Comment:
Where this file is used?

Reply:
It is not yet used it is just being created. 

> Create a new dummy screen called Temp. In Temp, you should display the new placeholder view. Add the logic to toggle the visibility of the placeholder view based on the screen size (mobile breakpoint).

Or Am I missing something here?

Reply:
no, all good then